mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -369,6 +409,7 @@
         </InfoArray>
     </CAbilResearch>
     <CAbilTrain id="GatewayTrain">
+        <InfoArray index="Train6" Time="32"/>
```

Comment:
Sentry build time reduced from 26.4 to 22.9 seconds (from 37 to 32 on normal speed)